CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+Changes to Guelph Coding Community (GCC) policy documents will be welcomed via Pull Request (PR) by members of the current club executive body.
+
+## Document Style
+
+For Markdown source documents, please adhere to the following guidelines:
+
+ - Begin each document with a YAML front-matter which defines page styling for Pandoc
```

Comment:
Maybe we can put an example? (idk what pandoc is :-/)

Reply:
Yep! Can do! I'll also link to Pandoc reference documents :)

Reply:
Hey @De-ma I've updated this document, can you let me know if it is more clear how to contribute now?

Reply:
Gonna resolve this.